packages/app/src/components/Root/Root.tsx

```diff
@@ -101,10 +102,19 @@ const getMenuItem = (menuItem: ResolvedMenuItem, isNestedMenuItem = false) => {
   );
 };
 
+const GlobalHeader = ({ mountPoints }: { mountPoints: MountPoints }) =>
+  mountPoints['application/header']?.map(({ Component, config }) => (
+    <Component key="global-header" {...config?.props} />
```

Comment:
If you map an array of mount points you should use a unique key for the component. You should also wrap all headers in an ErrorBoundary. The ErrorBoundary we could also add later, but then please create a follow-up story for that.